packages/permission-log-controller/LICENSE

```diff
@@ -0,0 +1,20 @@
+MIT License
```

Comment:
Hmm, this would be changing the existing license for this code. The extension uses this license: https://github.com/MetaMask/metamask-extension/blob/develop/LICENSE. So we should probably keep this license.

We may need to adjust the Yarn constraints to accommodate this, but I can advise on that further.